### ISSUE TITLE
literally no way to tell what tab is active

### ISSUE THREAD

**Issue by BishopCrypto** [warpdotdev/Warp#9173]
### Pre-submit Checks

- [x] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3AFEATURE) and there are no duplicates
- [x] I have [searched Warp docs](https://docs.warp.dev) and my feature is not there

### Describe the solution you'd like?

the new sidebar tab thing wastes a ton of space i don't want to waste to show redundant nonsense information.  i don't need the path.   now there's also no way to tell what tab you are in.  it has become 50% less useful as a result

can you tell which tab is active?  there is no way to do it!

<img width="672" height="2142" alt="Image" src="https://github.com/user-attachments/assets/9eeea30e-6865-485b-8ba2-595ab8c7b321" />

### Is your feature request related to a problem? Please describe.

can't tell what tab is active there's nothign to differentiate it from the other tabs.  the color changing from its previous color does not help you grok which one you are using at a glance.

### Operating system (OS)

macOS

### How important is this feature to you?

5 (Can't work without it!)

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by BishopCrypto** [comment]
You need to be able to tell at a glance.  Depending on colors you literally cannot see which one is highlighted.
